src/calibration/model.py

```diff
@@ -2,5 +2,30 @@
 
 
 class CalibrationModel():
+    mainLocales = dict(
+        en=dict(
+            window='Calibration',
+            btnNext='Next',
+            btnCancel='Cancel',
+            btnFinish='Finish',
+        ),
+        ru=dict(
+            window='Калибровка',
+            btnNext='Далее',
+            btnCancel='Отмена',
+            btnFinish='Завершить',
+        ),
+    )
+
```

Comment:
why do you want to use a separate place, not the common. Not the common approach?

Reply:
I intended to use separate place in order to keep the calibration staff as a separate module. Now it is very easy to make calibration to be a standalone program. Probably it is time to start mix it with a main program, do you confirm?

Reply:
not really, it still can be a separate module, in a separate folder with separate translations, but with the common parent class.

Reply:
Sorry, what common parent class do you mean? My idea was to import _src.locales_ module and use its _getLocale()_  function

Reply:
yes, let's create locale pkg. Here I meant that you can create a class with translations, not just dicts.